### PR TITLE
[11.0] [ADD] stock_request_analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This means to integrate stock_request to analytics. The analytic account is passed from the stock_request to the stock moves. That way we can account for stock_requests and classify them.

### Roadmap
The integration with stock_request_purchase. Done in #520

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
This is ready to review. It seems there is some load issue in runbot, I'm restarting the build.

File: stock_request_analytic/models/stock_request_order.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Creu Blanca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class StockRequestOrder(models.Model):
+    _inherit = 'stock.request.order'
+
+    analytic_count = fields.Integer(string='analytic count',
```
Comment (rousseldenis) [inline stock_request_analytic/models/stock_request_order.py]: No need of the string

File: stock_request_analytic/models/stock_request_order.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                                    compute='_compute_analytic_ids',
+                                    readonly=True)
+    analytic_account_ids = fields.One2many(
+        'account.analytic.account',
+        compute='_compute_analytic_ids',
+        string='Analytic Accounts',
+        readonly=True, copy=False)
```
Comment (rousseldenis) [inline stock_request_analytic/models/stock_request_order.py]: IMHO copy is not needed

File: stock_request_analytic/models/stock_request_order.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    @api.depends('stock_request_ids')
+    def _compute_analytic_ids(self):
+        for req in self.sudo():
+            req.analytic_ids = req.stock_request_ids.mapped(
```
Comment (rousseldenis) [inline stock_request_analytic/models/stock_request_order.py]: I think this field does not exist

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I attended your review and dis some minor fixes. Could you have a look?

**Review by LoisRForgeFlow — APPROVED** [review]
IMO this is ready :+1:

**Review by rousseldenis — APPROVED** [review]
Code review. @aheficent Maybe you could use README fragments?

**Pull request merged**